src/cloud-api-adaptor/docs/SecureComms

```diff
@@ -23,7 +23,7 @@ Change the `podvm/files/etc/systemd/system/agent-protocol-forwarder.service` to
 ExecStart=/usr/local/bin/agent-protocol-forwarder -kata-agent-namespace /run/netns/podns -secure-comms -kata-agent-socket /run/kata-containers/agent.sock $TLS_OPTIONS $OPTIONS
 ```
 
-You may also include additional Inbounds and Outbounds configurations to the Forwarder using the `-secure-comms-inbounds` and `-secure-comms-outbounds` flags.
+You may also include additional Inbounds and Outbounds configurations to the Forwarder using the `-secure-comms-inbounds` and `-secure-comms-outbounds` flags. See more details regarding Inbounds and Outbounds below.
```

Comment:
You are introducing this file at commit 9ca7f6b3dbed50ff6b694a65a4df50a102dd6291. There is no need to send another commit to fix typos. Please, fix the typos on the same commit you are introducing the file.

Reply:
I will squash the commits to resolve this and many similar issues in this PR.